javascript-objects/main.js

```diff
@@ -0,0 +1,35 @@
+var student = {
+  firstName: 'Jay',
+  lastName: 'Park',
+  age: 21
+};
+
+var fullName;
+fullName = student.firstName + ' ' + student.lastName;
```

Comment:
This isn't necessary, you can declare a variable and assign it a value on the same line.